push-subscription-management/server.js

```diff
@@ -0,0 +1,61 @@
+var webPush = require('web-push');
+// collection of endpoints stored in memory (in production it should be
+// a database)
+var subscriptions = [];
```

Comment:
That would add some complication, but it would be a workaround to https://bugzilla.mozilla.org/show_bug.cgi?id=1222428